tests/test_bgp_af.rb

```diff
@@ -376,6 +520,8 @@ def advertise_l2vpn_evpn(asn, vrf, af)
   end
 
   def test_advertise_l2vpn_evpn
+    # no equivalent for IOS XR
+    return if platform != :nexus
```

Comment:
Instead, please do:

`skip('L2VPN EVPN is unsupported on IOS XR') if platform == :ios_xr`